src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer  started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stop working");

```

Comment:
The message in the 'stopWork' method should be more informative and consistent with the 'doWork' method. It should specify that it is a Bulldozer that has stopped working and should use the same verb tense, e.g., 'Bulldozer has stopped working.'